docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
```

Comment:
<img width="128" alt="ss11" src="https://user-images.githubusercontent.com/60163283/140518653-694421f8-5435-43a5-b6b1-eedac1d7c869.PNG">
Can use :stock instead of instrument. in the explanation part can just say edit parameters for stock as an example. We never edit for in
strument type but always the child type